$NetBSD: patch-Makefile,v 1.1 2018/08/13 15:53:49 nia Exp $

Use native compiler for producing build executable, and
CFLAGS from pkgsrc.

--- Makefile.orig	2007-05-27 11:45:09.000000000 +0000
+++ Makefile
@@ -8,13 +8,10 @@ dsk2rom_e = dsk2rom
 bin2h_e   = bin2h
 endif
 
-CC = g++
-CFLAGS = -O2 -Wall -fomit-frame-pointer
-
 all: bin2h dsk2rom.h dsk2rom
 
 bin2h: bin2h.c
-	$(CC) $(CFLAGS) $^ -o $(bin2h_e)
+	$(NATIVE_CC) $^ -o $(bin2h_e)
 
 pletter/pletter.o: pletter/pletter.cpp pletter/pletter.h
 
@@ -22,7 +19,7 @@ dsk2rom.h: bin2h
 	$(shell ./${bin2h_e} dsk2rom dsk2rom.rom dsk2rom.h)
 
 dsk2rom: dsk2rom.c dsk2rom.h pletter/pletter.o
-	$(CC) $(CFLAGS) $^ -o $(dsk2rom_e)
+	$(CXX) $(CFLAGS) $(LDFLAGS) $^ -o $(dsk2rom_e)
 
 clean:
 	rm -f $(dsk2rom_e)
